demisto_sdk/commands/common/docker/dockerhub_client.py

```diff
@@ -437,7 +437,16 @@ def get_docker_image_tag_creation_date(
         """
         response = self.get_image_tag_metadata(docker_image, tag=tag)
         if creation_date := response.get("created"):
-            return datetime.strptime(creation_date, "%Y-%m-%dT%H:%M:%S.%fZ")
+            try:
+                return datetime.strptime(creation_date, "%Y-%m-%dT%H:%M:%S.%fZ")
+            except ValueError as e:
+                logger.debug(
+                    f"Failed initial parsing attempt for '{creation_date=}', docker image {docker_image}:{tag}. Exception: {e}"
+                )
+                logger.debug("Attempting to normalize timestamp and retry parsing.")
+                return datetime.strptime(
+                    iso8601_to_datetime_str(creation_date), "%Y-%m-%dT%H:%M:%S.%fZ"
+                )
```

Comment:
```suggestion
                return datetime.strptime(
                    iso8601_to_datetime_str(creation_date), "%Y-%m-%dT%H:%M:%S.%fZ"
                )
```
would work just as well 🤷 it's not like someone will ever view the logs errors.